crates/exec-wasmtime/src/loader/compiled/mod.rs

```diff
@@ -47,13 +45,15 @@ impl Loader<Compiled> {
                 File::Stdout { .. } => (Box::new(stdout()), FileCaps::all()),
                 File::Stderr { .. } => (Box::new(stderr()), FileCaps::all()),
 
-                File::Listen { port, prot, .. } => {
+                File::Listen {
+                    addr, port, prot, ..
+                } => {
                     let caps = FileCaps::FILESTAT_GET
                         | FileCaps::FDSTAT_SET_FLAGS
                         | FileCaps::POLL_READWRITE
                         | FileCaps::READ;
 
-                    let tcp = std::net::TcpListener::bind((Ipv4Addr::UNSPECIFIED, *port))?;
+                    let tcp = std::net::TcpListener::bind((&**addr, *port))?;
```

Comment:
Is there a way we can avoid the `&**`? Perhaps a `ref addr` on L49 could help at least a bit? Is the purpose of this to make a bitwise copy? (if so, shouldn't we use a method?)

Reply:
I can use `addr.as_str()`